packages/eslint-plugin/lib/rules/typescript/prefer-build-client-schema.js

```diff
@@ -0,0 +1,33 @@
+const {docsUrl} = require('../../utilities');
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Prefer buildClientSchema for schema building',
+      category: 'Best Practices',
+      recommended: true,
+      uri: docsUrl('typescript/prefer-build-client-schema'),
+    },
+    fixable: null,
+  },
+  create(context) {
+    function report(node) {
+      context.report({
+        node,
+        message: `Prefer buildClientSchema to buildSchema`,
```

Comment:
I wonder if we could even apply the fix here?

https://eslint.org/docs/developer-guide/working-with-rules#applying-fixes-1

Reply:
I was thinking about this, but due to not knowing how `buildSchema` could be used, and the differences in implementation, applying a standardized fix across `x` number of repos, with `x` implementations could get weird.